braid/residual.c

```diff
@@ -43,19 +44,45 @@ _braid_Residual(braid_Core        core,
    braid_Int        ilower   = _braid_GridElt(grids[level], ilower);
    braid_Real      *ta       = _braid_GridElt(grids[level], ta);
 
+   /* this decides whether Delta correction needs to be computed on this level,
+   *  which depends on from where the residual is called
+   */
+   braid_Int delta_correct;
+   delta_correct = _braid_CoreElt(core, delta_correct)
+                   &&  ( iter  >= _braid_CoreElt(core, delta_defer_iter) )
+                   &&  ( level >= _braid_CoreElt(core, delta_defer_lvl) )
+                   && !( calling_function == braid_ASCaller_Residual
+                      && level == _braid_CoreElt(core, delta_defer_lvl) );
```

Comment:
Why does this include both 
  level == _braid_CoreElt(core, delta_defer_lvl)
and 
  ( level >= _braid_CoreElt(core, delta_defer_lvl)
Is this a typo?